sys-libs/glibc/files/eblits/src_install.eblit

```diff
@@ -166,10 +166,21 @@ toolchain-glibc_src_install() {
 	# Clean out any default configs
 	rm -rf "${ED}"/etc
 
+	cd "${S}"
+
+	# Install misc network config files
+	insinto /usr/share/baselayout
+	doins nscd/nscd.conf || die
```

Comment:
It is the unmodified file from the glibc source.  We're not enabling the service by default, so I don't see a need to modify it at the moment.